test/keypress.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+import React from 'react';
+import test from 'ava';
+
+import {render, useKeypress} from '..';
+
+test.cb('letters work', t => {
+	t.plan(1);
+	const input = 'abcdefghijklmnopqrstuvwxyz';
+	// Process.stdin.isTTY = false;
+	const hits = '';
+	const KeypressTest = () => {
+		useKeypress((str, _) => {
+			hits.push(str);
+			if (str === 'z') {
+				t.deepEqual(hits, input);
+				t.end();
+			}
+		});
+		return null;
+	};
+
+	render(<KeypressTest/>);
+	for (const letter of input.split('')) {
+		process.stdin.push(Buffer.from(letter));
```

Comment:
It's more correct to use `process.stdin.write()`. Then you don't need to wrap it in a Buffer either.